### PR TITLE
scheduler: enable reserve plugin and cache by default

### PR DESCRIPTION
**Pull request by ffromani**
Time to enable by default the cacheResyncPeriod.
We will handle the compatibility just requiring in the release notes a minimum version of the scheduler plugin.

### CODE REVIEW

**Comment by ffromani** [comment]
/hold

need to check the tests

**Comment by ffromani** [comment]
/hold cancel

ready for review

File: controllers/numaresourcesscheduler_controller.go [hunk 1]
```diff
@@ -103,15 +103,13 @@ func (r *NUMAResourcesSchedulerReconciler) Reconcile(ctx context.Context, req ct
...
 
 	result, condition, err := r.reconcileResource(ctx, instance)
-	if condition != "" {
-		// TODO: use proper reason
-		reason, message := condition, messageFromError(err)
-		if err := r.updateStatus(ctx, instance, condition, reason, message); err != nil {
-			klog.InfoS("Failed to update numaresourcesscheduler status", "Desired condition", condition, "error", err)
-		}
+
+	reason := condition // TODO: use proper reason
```
Comment (swatisehgal) [inline controllers/numaresourcesscheduler_controller.go:111]: Just curious: is `reason` meant to be with more detailed version of a `condition`?

**Pull request merged**